mlflow/store/db/utils.py

```diff
@@ -81,9 +82,12 @@ def make_managed_session():
         except MlflowException:
             session.rollback()
             raise
+        except sqlite3.IntegrityError as e:
```

Comment:
This should be `sqlalchemy.exc.IntegrityError`

Reply:
it works!